RealmSwift/ObjectiveCSupport+AnyRealmValue.swift

```diff
@@ -50,6 +50,12 @@ public extension ObjectiveCSupport {
             return u as NSUUID
         case let .object(o):
             return o
+        case let .dictionary(d):
+            let rlmDictionary = Map<String, AnyRealmValue>._rlmFromObjc(d)?.rlmDictionary
```

Comment:
Why is this wrapping the object in the Swift type and then immediately rereading the thing it just wrapped?